html/top-stories.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Top Stories</title>
+        <link rel="stylesheet" href="../css/style.css">
+        <link rel="stylesheet" href="../css/top-stories.css">
+        <link href="https://fonts.googleapis.com" rel="preconnect">
+        <link crossorigin href="https://fonts.gstatic.com" rel="preconnect">
+        <link href="https://fonts.googleapis.com/css2?family=Rubik:wght@300;400;700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <div class="main-container">
+            <header class="header">
+                <div class="header-container">
+                    <div class="header-logo">
+                        <span class="login-header-text-first"
+                              style="color: #FFFFFF;
```

Comment:
Please avoid inline style

Reply:
![image](https://user-images.githubusercontent.com/34778670/167312085-9c891d5d-063f-4aaa-a377-076d37750240.png)


I am fix inline style in html-code. I would be heppy to write coments in pull request, but i do not khow English well

Reply:
Do write comments in English! Even bad English is better then No English at all :)